src/server.h

```diff
@@ -1404,6 +1409,9 @@ struct redisServer {
     int io_threads_do_reads;    /* Read and parse from IO threads? */
     int io_threads_active;      /* Is IO threads currently active? */
     long long events_processed_while_blocked; /* processEventsWhileBlocked() */
+    int enable_protected_configs;    /* Enable the modification of protected configs */
+    int enable_debug_cmd;            /* Enable DEBUG commands */
+    int enable_module_cmd;           /* Enable MODULE commands */
```

Comment:
Worth mentioning these are enum values, it's not obvious that they aren't booleans.